src/cloud-api-adaptor/podvm/Dockerfile.podvm_builder.rhel

```diff
@@ -50,7 +50,7 @@ RUN chmod a+x /usr/local/bin/yq && \
     curl https://dl.google.com/go/go${GO_VERSION}.linux-${YQ_ARCH}.tar.gz -o go${GO_VERSION}.linux-${YQ_ARCH}.tar.gz && \
     rm -rf /usr/local/go && tar -C /usr/local -xzf go${GO_VERSION}.linux-${YQ_ARCH}.tar.gz && \
     rm -f go${GO_VERSION}.linux-${YQ_ARCH}.tar.gz
-ENV PATH="/usr/local/go/bin:${PATH}"
+ENV PATH="/usr/local/go/bin:/usr/local/bin${PATH}"
```

Comment:
You're missing the `:` separator between `/usr/local/bin` and `${PATH}`.

Reply:
will fix, saw typos in the commit messages so a good chance to fix those too

Reply:
fixed.